pkg/goods/materializer.go

```diff
@@ -76,6 +76,19 @@ func (m *Materializer) CalicoNetworkManagerConfig() error {
 	return nil
 }
 
+// SysctlConfig writes the embedded sysctl config to the /etc/sysctl.d directory.
+func (m *Materializer) SysctlConfig() error {
```

Comment:
Would it be worth adding a unit test for this method?

Reply:
Added.